src/index.html

```diff
@@ -1,14 +1,22 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body class="page__body">
+    <main class="stopwatch">
+      <div class="stopwatch__hands hands">
+        <div class="hands__pin"></div>
+
+        <div class="hands__seconds"></div>
+        
+        <div class="hands__minutes"></div>
```

Comment:
it's better to add an empty line between _multiline_ siblings not _oneline_ ones
```suggestion
        <div class="hands__pin"></div>
        <div class="hands__seconds"></div>
        <div class="hands__minutes"></div>
```